Public/Src/Engine/Scheduler/Artifacts/FileContentManager.cs

```diff
@@ -1635,6 +1636,15 @@ private void PopulateArtifactsToMaterialize(PipArtifactsState state, bool? allow
                             state.MaterializedDirectoryContents.Add(file);
                         }
 
+                        if (!OperatingSystemHelper.IsUnixOS)
+                        {
+                            if (FileSystemWin.IsPathReparseDirectory(file.Path.ToString(Context.PathTable)))

```

Comment:
>IsPathReparseDirectory [](start = 46, length = 22)

This should not be here. The file may not even be on disk at this point. What is the intent of this line?

Reply:
Removed. This might be needed if we consider the consumer pip who might seal the directory after the create symlink pip generate it. But at this point since we don't care the functionality of the symlink directory, I will remove it.